src/components/Highlight.vue

```diff
@@ -29,27 +28,11 @@ export default {
   },
   computed: {
     innerHTML() {
-      const attributeValue =
-        getPropertyByPath(
-          this.hit,
-          `_highlightResult.${this.attribute}.value`
-        ) || '';
-
-      if (process.env.NODE_ENV !== 'production' && attributeValue === '') {
-        warn(
```

Comment:
We don't have the warning anymore, but it shouldn't be the responsibility of Vue InstantSearch. If we want to warn it should be done inside InstantSearch.js. I don't think it has a huge impact to remove it from Vue at the moment.